package.json

```diff
@@ -27,11 +27,11 @@
   "main": "src/index.js",
   "homepage": "https://github.com/electron-userland/electron-spellchecker",
   "dependencies": {
-    "@nornagon/cld": "^2.4.9",
     "@felixrieseberg/spellchecker": "^4.0.10",
     "bcp47": "^1.1.2",
+    "cld": "^2.5.1",
     "debug": "^4.1.1",
-    "electron-remote": "^1.3.0",
+    "electron-remote": "https://github.com/aabuhijleh/electron-remote/tarball/master",
```

Comment:
I don't think that solution is quite enough to fix things here - it's worthy hack, but I'd rather have a real fork than pure GitHub repo. 

Would you mind publishing your fork as an (immutable) fork to npm?

Reply:
I agree with you that this is a hack. Maybe "electron-remote" should be replaced all together (#152).

For now, I tested this change on my Electron app on Windows and it's working well.